### ISSUE TITLE
[feature] Always show links for current page on top.

### ISSUE THREAD

**Issue by rigrig** [tvler/streetpass#48]
I often wonder if there are any fediverse links on a page I've visited before. The only way to check seems to be scrolling through the entire list of matches (172 so far) and check if the site is present.   
If matches for the current page (if any) were on top of the list, it would make things easier.

**Comment by ploum** [comment]
Indeed. At first, I thought there was a bug as my account was not showed anymore when browsing my own website. It took me a while to realize that streetpass only display an account the first time you see it.

IMHO, this is confusing and this doesn’t make sense. I’m interested by the accounts of the latest pages I’ve seen. The fact that I already visited that website several days/weeks ago is completely irrelevant. I fail to see any usecase where you would want to *not* see an account because you already saw it before. On the contrary, I’m currently starting to use streetpass as a shortcut to see mastodon news from people I know: I go to their website and immediately go to their mastodon account. It is faster than opening mastodon then searching.  But this usecase works only for the first days.


You could say that I could reset streetpass but I’m interested to keep mastodon addresses from websites visited before. It is a really nice sociological experiment.


I would then argue for an option (or even to make it the default) to display account in the order they were seen for the last time, not seen for the first time.
